src/core/scenario.jl

```diff
@@ -183,6 +184,13 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
         storage_structs["ColdThermalStorage"] = ColdThermalStorage(params, financial, settings.time_steps_per_hour)
     end
     storage = Storage(storage_structs)
+    dc_coupled_pv = any([pv.dc_coupled_with_storage && pv.max_kw > 0 for pv in pvs])
+    if dc_coupled_pv && isempty(storage.types.dc_coupled)
```

Comment:
I do think just defaulting this storage attribute to true and not exposing it as a user input might be preferable, so that a user doesn't have to change two inputs to do one thing.

Reply:
Ok sounds good